QuickOSM/test/test_preset_parser.py

```diff
@@ -0,0 +1,176 @@
+import xml.dom.minidom as xml
+
+from qgis.testing import unittest
+
+from QuickOSM.core.parser.preset_parser import PresetsParser
+
+__copyright__ = 'Copyright 2021, 3Liz'
+__license__ = 'GPL version 3'
+__email__ = 'info@3liz.org'
+
+
+class TestQuickOSMWidget(unittest.TestCase):
+
+    def setUp(self):
+        self.maxDiff = None
+
+    def test_parser(self):
```

Comment:
Can you explain each tests ? THe main goal of this XML string ?
Maybe split in different `def test_parser_single_key` etc ...